Systematics/python/flashggMuonSystematics_cfi.py

```diff
@@ -211,7 +216,14 @@ def SetupMuonScaleFactors( process , id_file_name, id_lowpt_file_name, iso_file_
         MUON_ISO_ScaleFactors[ iso ] = MuonSF_JSONReader( iso_file_name , "NUM_" + iso , "", "", extendPtValISO)
 
     if id_name in MUON_ID_ScaleFactors.keys():
-        process.flashggMuonSystematics.SystMethods.append( MUON_ID_ScaleFactors[id_name].GetPSet(str("Muon" + id_name + "IDWeight")) )
+	if (code == 0):
```

Comment:
I think here one could use something like:
1) remove code
2) setattr(process, "flashggMuonSystematics"+id_name, flashggMuonSystematics.clone())
    getattr(process, "flashggMuonSystematics"+id_name).SystMethods.append( ... )
removing the need of an hardcoded swith